src/PendingObjects/TestCall.php

```diff
@@ -157,6 +170,40 @@ public function skip($conditionOrMessage = true, string $message = ''): TestCall
         return $this;
     }
 
+    /**
+     * Compare two given PHP versions.
```

Comment:
We should add typehints here. 👍🏻
```suggestion
     * Compare two given PHP versions.
     *
     * @param string|array $compared
     * @param string|array $version
```

I feel like it's probably better to use `version_compare()` in this method.

Reply:
I've used `version_compare()`